### PR TITLE
[WIP] Drop support for Ruby 2.3

### LINKED ISSUE #184: Drop support for ruby 2.3
Ruby 2.3 is EOL.  Drop support for it.

### PR DESCRIPTION
**Pull request by EmilyMB**
* Drops support for Ruby 2.3 (which reached its end of life on March 31, 2019). Fixes #184.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ end
 
 # This is a minimal set of Gems required to build on travis
 group :test do
-  gem 'bundler'
+  gem 'bundler', '~>1.16'
```
Comment (olbrich) [inline Gemfile]: Why did you need to do this?

File: .travis.yml [hunk 2]
```diff
@@ -6,7 +6,6 @@ rvm:
   - 2.6
   - 2.5
   - 2.4
-  - 2.3
```
[the remark this answers is not in the document]
Comment (mathieujobin) [reply .travis.yml]: or wait until test fails and we decide we won't bother fixing...

**Comment by olbrich** [comment]
No longer necessary
